src/theme/colors.scss

```diff
@@ -0,0 +1,4 @@
+$lightGrey: #535c684d;
```

Comment:
This name is a little inconsistent. Please consider a change to either this or the rest of the names.